### PR TITLE
Updated Notebook.

### PR DESCRIPTION
**Pull request by scottdangelo**
Add quality monitoring section.
No need for spark_creds anymore.
Add method to run on demand.
Remove empty cells.
Add instructions for `wml_client` field `binding_uid`.

### CODE REVIEW

File: notebooks/DataMart.ipynb [hunk 1]
```diff
@@ -618,21 +616,46 @@
...
+   "source": [
+    "subscription.feedback_logging.show_table()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Feedback records are used to evaluate our model."
+    "### 3.4 Enable Fairness monitor for best hurt drug asset"
```
Comment (markstur) [inline notebooks/DataMart.ipynb]: hurt drug?

File: notebooks/DataMart.ipynb [hunk 2]
```diff
@@ -329,7 +291,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can bind you Watson Machine Learning instance to previously created data mart."
+    "You can bind you Watson Machine Learning instance to previously created data mart.\n",
```
Comment (markstur) [inline notebooks/DataMart.ipynb]: s/you Watson/your Watson/

**Review by markstur — CHANGES_REQUESTED** [review]
see nits

File: notebooks/DataMart.ipynb (hunk 1 above)
Comment (scottdangelo) [reply notebooks/DataMart.ipynb]: done, thx.

File: notebooks/DataMart.ipynb (hunk 2 above)
Comment (scottdangelo) [reply notebooks/DataMart.ipynb]: done, thx.

**Review by markstur — APPROVED** [review]

**Pull request merged**